### DISCUSSION TITLE
🧰 ⛓️ ⚡ Feedback

### DISCUSSION THREAD

**Discussion by brunopgalvao** [r0gue-io/pop-cli#295] [General]
We want to build the most powerful CLI tool for Polkadot appchain & smart contract development.

Let [us](https://github.com/r0gue-io) know how we can make Pop CLI the most powerful CLI tool in the blockchain ecosystem.

Some guiding questions:
1. What would make you use this tool in your daily parachain (or smart contract) development?
2. What would make this tool indispensable for you for daily development?
3. What would really help you if it were added as a feature to Pop CLI?

Bonus Question:
- What would make this tool the `#1` CLI tool in the blockchain ecosystem? (think [hardhat](https://hardhat.org), [Solana IDE](https://beta.solpg.io), [Remix](https://remix-project.org), [ignite](https://ignite.com/))

**Comment by shawntabrizi** [comment]
- Have you optimized for the "starting from scratch" scenario?
     - When was the last time you tried pop-cli on a brand new computer with nothing installed?
     - How many steps / commands / tools does the user need to use to be able to get started?
     - Can pop-cli remove all those additional steps?
- Have you integrated other key tools for maintaining a project past just getting started?
    - https://github.com/ggwpez/zepter
    - https://github.com/paritytech/psvm

A bunch of ideas for a Polkadot CLI here: https://github.com/paritytech/polkadot-sdk/issues/5192
